opentracing/ext/tags.py

```diff
@@ -37,6 +37,11 @@
 SPAN_KIND_RPC_SERVER = 'server'
 
 # ---------------------------------------------------------------------------
+# ERROR indicates whether a Span ended in an error state.
+# ---------------------------------------------------------------------------
+ERROR = 'error'
```

Comment:
thanks!

in the "no good deed goes unpunished" dept, would you be willing to add the other missing tags from https://github.com/opentracing/specification/blob/master/data_conventions.yaml? Esp the new database ones.